### PR TITLE
Do not minify Pillow's libraries

### PR DESCRIPTION
**Pull request by jotamartos**
Pillow libraries fail when stripping them. We should avoid stripping them to make the solutions work.

Related PR: #246

Signed-off-by: Jota Martos <jotamartos@vmware.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "blacksmith",
-  "version": "3.7.3",
+  "version": "3.7.4",
```
Comment (marcosbc) [inline package.json:3]: Make sure to rebase, since 3.7.4 was released a bit less than an hour ago.

Comment (jotamartos) [reply package.json]: Thanks! :)

**Review by marcosbc — APPROVED** [review]
LGTM!

**Comment by marcosbc** [comment]
Let's see if the automated tests succeed, it seems to have spotted an error, hopefully it something transient.

**Pull request merged**